superset/db_engine_specs/base.py

```diff
@@ -77,7 +83,7 @@ class TimeGrain(NamedTuple):  # pylint: disable=too-few-public-methods
 class TimestampExpression(
     ColumnClause
 ):  # pylint: disable=abstract-method,too-many-ancestors,too-few-public-methods
-    def __init__(self, expr: str, col: ColumnClause, **kwargs):
+    def __init__(self, expr: str, col: ColumnClause, **kwargs: Any) -> None:
```

Comment:
can't kwargs be a `Dict[String, Any]`? Maybe we should define a global `Kwargs` type that matches this

Reply:
PEP484 says to type `**kwargs` and `*args` as the type of the arguments, not the type of `kwargs` (`dict`)  and `args`(`tuple`).